gpustat/core.py

```diff
@@ -337,6 +341,27 @@ def jsonify(self):
         return o
 
 
+class LostGPU(GPUStat):
```

Comment:
Can we add an attribute like `is_available` in the `GPUStat` class? So we can determine whether this GPU is available by a simple boolean value.  
In the current implementation, we need to use `import LostGPU; if isinstance(LostGPU): XXX`, which exposes the underlying details to the final users.

Reply:
Sounds good. Will add a method `available()`.